README.md

```diff
@@ -445,6 +445,8 @@ Supported options:
 - `dir` (`string`) where the Next project is located - default `'.'`
 - `quiet` (`bool`) Hide error messages containing server information - default `false`
 
+Then, change your `build` script to `NODE_ENV=production node server.js`.
```

Comment:
Since you still have to `next build` when using the custom server.

https://github.com/zeit/next.js/blob/master/examples/custom-server/package.json#L5